Templates/Class Template/q6.cpp

```diff
@@ -0,0 +1,26 @@
+// WAP to compare 2 values using a function template and display the largest value. Check your program for char, int, and float type data.
+
+#include <iostream>
+using namespace std;
+
+// Function template to find the maximum of two values
+template <class T>
+T findMax(T a, T b) {
+    return (a > b) ? a : b;
+}
+
+int main() {
+    // Comparing two char values
+    char char1 = 'a', char2 = 'z';
```

Comment:
can you make it dynamic?

Reply:
> can you make it dynamic?

Yes sure, I will update it and push it again.